backend/src/routes/api/storage/index.ts

```diff
@@ -0,0 +1,47 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { createMinioClient, getObjectStream } from './storageUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get('/:namespace/:bucket', async (request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
Technically the Request & Reply are generics

Reply:
fixed, i think